Tests/SentryTests/Helper/SentryCurrentDateTests.swift

```diff
@@ -2,6 +2,15 @@ import XCTest
 @testable import Sentry
 
 class SentryCurrentDateTests: XCTestCase {
+    
+    func testSetNoCurrentDateProvider() {
+        let expected = Date.init()
+        
+        let actual =  CurrentDate.date()
+        
+        XCTAssertEqual(expected.timeIntervalSinceReferenceDate,
```

Comment:
M: At this precision can this become flakey?

What about we just compare if `actual` is **after** `expected` here, and then get a new `Date.init()` and test that `actual` is now before this new date? This way the test isnt' flakey, we're not really testing precision but we make sure `CurrentDate.date()` returns a date that makes sense with the OS.

Variable names would need to be renamed then.

Reply:
Very good point. Thanks for this nice suggestions 🥇